components/ide/code/leeway.nightly.Dockerfile

```
@@ -0,0 +1,101 @@
+# Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+FROM gitpod/openvscode-server-linux-build-agent:focal-x64 as code_builder
+
+ENV TRIGGER_REBUILD 1
```

Maybe this syntax also works, just struck me when reading

```suggestion
ENV TRIGGER_REBUILD=1
```